src/main/webapp/app/course/manage/course-management-tab-bar/course-management-tab-bar.component.html

```diff
@@ -79,6 +79,12 @@
                     <span class="tab-link-text" jhiTranslate="entity.action.scores">Scores</span>
                 </a>
             }
+            @if (course.isAtLeastInstructor) {
+                <a class="tab-link" [routerLink]="['/course-management', course.id, 'build-queue']" routerLinkActive="active">
+                    <fa-icon [icon]="faGears"></fa-icon>
```

Comment:
The gear icon is quite general and very familiar with settings in general. What do you think about using the faList icon instead
![image](https://github.com/ls1intum/Artemis/assets/78978542/816f77fa-b5fd-423b-855a-338187c173b9)